sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/action/ActionsImpl.scala

```diff
@@ -207,7 +203,7 @@ private[javasdk] final class ActionsImpl(
             val decodedPayload = service.messageCodec.decodeMessage(
               in.payload.getOrElse(throw new IllegalArgumentException("No command payload")))
             val effect = service.factory
-              .create(creationContext)
+              .create(context.asInstanceOf[ActionCreationContext])
```

Comment:
Why is the cast needed?

Reply:
Not needed anymore. Good catch. It was needed before https://github.com/lightbend/kalix-jvm-sdk/pull/2007#discussion_r1481330135

Reply:
removed